src/zlog.h

```diff
@@ -276,4 +278,74 @@ typedef enum {
 }
 #endif
 
+struct CPPZLOG{
```

Comment:
Interface used in CPP should be seperated to a new head avoiding include CPP standard header in C projects